### PR TITLE
Created new field proxyHost in entity Login and was show in section SSH

### PR DESCRIPTION
**Pull request by gubarev-bizz**

### CODE REVIEW

**Comment by mcfedr** [comment]
I know I wrote in the task add proxy host as a field, but actually it needs to be a whole new Login object, I think we need to have a One-to-One link to another Login object that is the proxyLogin

**Comment by gubarev-bizz** [comment]
Окей я понял. Сейчас исправлю. Я там запушил task_10298_1

2015-09-14 15:01 GMT+03:00 Fred Cox notifications@github.com:

File: src/AppBundle/Entity/Login.php [hunk 1]
```diff
@@ -57,6 +57,15 @@ class Login
     private $hostname;
 
     /**
+     * @var string
+     *
+     * @ORM\OneToOne(targetEntity="Login")
+     * @ORM\JoinColumn(name="proxyHost", referencedColumnName="id")
```
Comment (mcfedr) [inline src/AppBundle/Entity/Login.php]: Dont need JoinColumn

File: src/AppBundle/Entity/Login.php [hunk 2]
```diff
@@ -212,6 +221,23 @@ public function getHostname()
     }
 
     /**
+     * @return string
```
Comment (mcfedr) [inline src/AppBundle/Entity/Login.php]: Change this to @return Login

File: src/AppBundle/Entity/Login.php [hunk 3]
```diff
@@ -212,6 +221,23 @@ public function getHostname()
...
+     */
+    public function getProxyHost()
+    {
+        return $this->proxyHost;
+    }
+
+    /**
+     * @param string $proxyHost
+     */
+    public function setProxyHost($proxyHost)
```
Comment (mcfedr) [inline src/AppBundle/Entity/Login.php]:
```
setProxyHost(Login $proxyHost = null)
```

File: app/DoctrineMigrations/Version20150914144503.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE login ADD proxyHost VARCHAR(255) DEFAULT NULL');
```
Comment (mcfedr) [inline app/DoctrineMigrations/Version20150914144503.php]: This is clearly wrong

**Pull request merged**